main.tf

```diff
@@ -0,0 +1,88 @@
+terraform {
+  required_providers {
+    azurerm = {
+      source  = "hashicorp/azurerm"
+      version = ">=3.0.0"
+    }
+    azuread = {
+      source  = "hashicorp/azuread"
+      version = ">=2.47.0"
+    }
+  }
+}
+
+provider "azurerm" {
+  skip_provider_registration = true
+  features {}
+}
+
+data "azurerm_storage_account" "worklytics" {
+  name                = var.storage_account_name
+  resource_group_name = var.resource_group_name
+}
+
+# Create a storage container (https://<account>.blob.core.windows.net/<container>)
+# https://registry.terraform.io/providers/hashicorp/azurerm/latest/docs/resources/storage_container
+resource "azurerm_storage_container" "worklytics" {
+  name                  = "${var.resource_name_prefix}container"
+  storage_account_name  = var.storage_account_name
+  container_access_type = "blob"
```

Comment:
:yellow_circle: I think this needs to be `private`, as we only want to allow access to the container with authenticated requests (but please test it, as i am not sure)